awx/main/tests/unit/test_tasks.py

```
@@ -1303,10 +1303,10 @@ def get_cred():
             if credential:
                 credential.credential_type.inject_credential(credential, env, safe_env, [], private_data_dir)
 
-        env["VMWARE_USER"] == "bob",
-        env["VMWARE_PASSWORD"] == "secret",
-        env["VMWARE_HOST"] == "https://example.org",
-        env["VMWARE_VALIDATE_CERTS"] == "False",
+        (env["VMWARE_USER"] == "bob",)
+        (env["VMWARE_PASSWORD"] == "secret",)
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "VMWARE_PASSWORD"

(S105)